plugins/llvm/src/main/java/org/qbicc/plugin/llvm/LLVMCompileStage.java

```diff
@@ -108,14 +22,7 @@ public void accept(final CompilationContext context) {
 
         Iterator<Path> iterator = llvmState.getModulePaths().iterator();
         context.runParallelTask(ctxt -> {
-            LlcInvoker llcInvoker = createLlcInvoker(ctxt);
-            OptInvoker optInvoker = createOptInvoker(ctxt);
-            CCompilerInvoker ccInvoker = createCCompilerInvoker(ctxt);
-
-            if (llcInvoker == null || optInvoker == null || ccInvoker == null) {
-                return;
-            }
-
+            LLVMCompiler compiler = new LLVMCompiler(context, isPie);
```

Comment:
What is the benefit of this refactor, is it mainly code organization or does some new functionality get enabled as a result?

Reply:
Mainly code organization.